modules/library/jdbc/src/main/java/org/geotools/jdbc/PreparedStatementSQLDialect.java

```diff
@@ -119,16 +119,23 @@ public abstract void setGeometryValue(
      *
      * @param value the value.
      * @param binding The class of the value.
+     * @param att Optionally the descriptor of the respective attribute to provide native type
+     *     information.
      * @param ps The prepared statement.
      * @param column The column the value maps to.
      * @param cx The database connection.
      */
     public void setValue(
-            Object value, Class binding, PreparedStatement ps, int column, Connection cx)
+            Object value,
+            Class<?> binding,
+            AttributeDescriptor att,
+            PreparedStatement ps,
+            int column,
+            Connection cx)
```

Comment:
This is an API change in an existing method that might have been overridden in implementations outside of GeoTools code base. Can you talk about this change in ``docs/user/welcome/upgrade.rst`` for the 32.x series?

As an API break, this bit will prevent backport to the stable/maintenance series.

Reply:
Yes, I'm happy to do that.